pkg/limayaml/validate.go

```diff
@@ -315,6 +318,21 @@ func Validate(y *LimaYAML, warn bool) error {
 	if warn {
 		warnExperimental(y)
 	}
+
+	// Validate Param settings
```

Comment:
It might be useful to include tests in `validate_test.go` for catching or not catching these errors.

Reply:
I've added tests for these validations, and for the `probe.script` check above.